javascript.js

```diff
@@ -0,0 +1,20 @@
+// set variavles
+var user = []; //inline coment
```

Comment:
Typo inside comment in word `coment`. Possible corrections:  `comment` (37), `moment` (30), `cement` (26)